.circleci/config.yml

```diff
@@ -7,14 +7,13 @@ jobs:
   build:
     <<: *defaults
     docker:
-      - image: golang:1.10-stretch
+      - image: golang:1.13
```

Comment:
Did you remove `-stretch` intentionally? `1.13-stretch` image seems to be smaller than `golang:1.13`
```
golang                      1.13-stretch        b130f0a1c39b        2 weeks ago         763MB
golang                      1.13                272e3f68338f        2 weeks ago         803MB
```

Reply:
!

thanks! I'm gonna update it. Also I'll migrate to GitHub action from circleci!